### ISSUE TITLE
modeldoc: resize per axis

### ISSUE THREAD

**Issue by gvarados1** [Facepunch/sbox-issues#19]
Resize per axis is useful for fitting imported modular models to the source grid.

This model has been resized on the y axis. We can do this in hammer but not in modeldoc (I think?)
![image](https://user-images.githubusercontent.com/39461072/115943414-740bee80-a475-11eb-957f-359f8a51ef22.png)

**Comment by garrynewman** [comment]
I don't think this is possible. Internally, for sceneobjects, I think scale is just a float.
